### PR TITLE
[Hub Generated] Review request for Microsoft.KubernetesConfiguration to add version preview/2019-11-01-preview

### PR DESCRIPTION
**Pull request by NarayanThiru**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/NarayanThiru/azure-rest-api-specs/dev-kubernetesconfiguration-Microsoft.KubernetesConfiguration-2019-11-01-preview...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by NarayanThiru** [comment]
We are removing the 3 enum values for ComplianceState property, as the agent is now sending additional values other than these three.  Since we anticipate more values to be returned, we want to remove these preset values.  Since this property has modelAsString set to true, we believe this is not a breaking change - as specified in the API revision guidelines.

**Comment by NarayanThiru** [comment]
Folks, there is some urgency to this, as we are targeting to release to Public Preview on the 11th of May.  We are taking these changes in response to JasonZ review feedback ,we received just couple of days ago.  I have added new enum values so as to not break the API.  As per guidelines, since the property is declared as modelAsString, add new values is not considered breaking change.  Can you pl. help review?  I requested @ArcturusZhang to be added, as he has prior context on this spec.

File: specification/kubernetesconfiguration/resource-manager/readme.go.md [hunk 1]
```diff
@@ -8,12 +8,12 @@ go:
   clear-output-folder: true
 ```
 
-### Go multi-api
+### Go multi-api
```
Comment (allenjzhang) [inline specification/kubernetesconfiguration/resource-manager/readme.go.md:11]: nit. With trailing whitespace removed, this file would not be in the PR.

File: specification/kubernetesconfiguration/resource-manager/readme.md [hunk 2]
```diff
@@ -78,27 +78,26 @@ See configuration in [readme.typescript.md](./readme.typescript.md)
 
 See configuration in [readme.csharp.md](./readme.csharp.md)
 
-## Multi-API/Profile support for AutoRest v3 generators 
+## Multi-API/Profile support for AutoRest v3 generators
```
Comment (allenjzhang) [inline specification/kubernetesconfiguration/resource-manager/readme.md:81]: I do not see meaningful changes in this file. Can you correct the trailing spaces so a clean diff can be reviewed. Thanks!

**Review by allenjzhang — CHANGES_REQUESTED** [review]
Once these issues are fixed, I can override the breaking change check failure as this is preview version of API. 
Please add label ReadyForSDKReview once you have made the fix to signal handover. Thanks!

**Comment by NarayanThiru** [comment]
@allenjzhang, I've made the changes.  Can you please merge the PR?

**Review by allenjzhang — APPROVED** [review]

**Review by NarayanThiru — COMMENTED** [review]
Added new values for the ComplianceState enum; and removed addl. whitespace

**Pull request merged**